content/en/cloud_cost_management/aws.md

```diff
@@ -147,7 +147,7 @@ Using Account Filtering requires an AWS management account. You can configure ac
 
 #### Configure account filters for a new account
 
-Navigate to [**Cloud Costs** > **Accounts**][18] and click **Add Account** on the AWS account card.
+Navigate to [**Cloud Cost** > **Accounts**][18] and click **Add Account** on the AWS account card.
```

Comment:
```suggestion
Navigate to [**Cloud Cost** > **Settings**, select **Accounts**][18], and then click **Add Account** on the AWS account card.
```

Reply:
changing it because the current path (**Cloud Cost** > **Accounts**) confused me. I was looking for **Accounts** in the navigation menu and it's not there. But **Settings** is.